plugin/pkg/auth/authenticator/password/passwordfile/passwordfile.go

```diff
@@ -60,9 +62,15 @@ func NewCSV(path string) (*PasswordAuthenticator, error) {
 			info:     &user.DefaultInfo{Name: record[1], UID: record[2]},
 			password: record[0],
 		}
+		if _, exist := users[obj.info.Name]; exist {
+			isDuplicate = true
```

Comment:
I just wanna keep the same warning with token_file, Is that OK to show the secrets in logs?